buddyboss/buddyboss-platform
# #2974 - Fix undefined function symlink() error when server doesn't support symlink

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2974.

### How to test the changes in this Pull Request:

1. Make sure to test it on the server that doesn't support symlink.
2. Use platform plugin version 1.7.6 above.
3. Activate the plugin and see the error.
4. If ever you see no error when activating the plugin, try deactivating the activity component and activate it back.

### Proof Screenshots or Video



### Other information:




### Changelog entry

>  Fix undefined function symlink() error when server doesn't support symlink
